### DISCUSSION TITLE
Pythonnet supports up to python 3.11

### DISCUSSION THREAD

**Discussion by LexiconCode** [amruthvvkp/flaui-uiautomation-wrapper#38] [General]
Hi, Looks like pythonnet supports up to python 3.11
https://github.com/pythonnet/pythonnet/releases

**Comment by amruthvvkp** [comment]
Thanks @LexiconCode, We've been working for a while to publish our first major release - v1.0.0. I've released a patch today that supports all Python versions between 3.7 and 3.11 - https://github.com/amruthvvkp/flaui-uiautomation-wrapper/releases.
